pkg/job/util.go

```diff
@@ -0,0 +1,31 @@
+package job
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+
+func SafeStringStripper(s string) string {
+	rChars := SafeAnnotationRegex()
+	return rChars.ReplaceAllString(s, "")
+}
+
+func IsSafeAnnotation(s string) bool {
+	matches := SafeAnnotationRegex().FindIndex([]byte(s))
+	return matches == nil
+}
+
+func SafeAnnotationRegex() *regexp.Regexp {
+	regexString := "A-Za-z0-9._~!:@,;+-"
+
+	file, _ := os.ReadFile("../../pkg/config/all_emojis.txt")
```

Comment:
does this work when the binary is shipped without the source code?

Reply:
also why are we thinking about emojis at all? i don't get it

Reply:
honestly just for fun - let people label their jobs with emojis?